src/main/java/io/qameta/allure/maven/AllureServeMojo.java

```diff
@@ -22,6 +22,24 @@
 @Mojo(name = "serve", defaultPhase = LifecyclePhase.SITE)
 public class AllureServeMojo extends AllureGenerateMojo {
 
+    /**
+     * Serve timeout parameter in seconds.
+     */
+    @Parameter(property = "allure.serve.timeout", defaultValue = "3600")
+    private String serveTimeout;
```

Comment:
I don't like this to be a string as well as new introduced report timeout to be a string

Reply:
@baev I send the patch on `allure@qameta.io` with `gh-59-fix-proposition` in email subject.

Reply:
@Coder1304 still got no email from you, could you please resend it?

Reply:
@baev I've send it again to `allure@qameta.io` with `gh-59-fix-proposition` in the subject. And as a reply for the first mail I send you.